### PR TITLE
Define GlobalTag to collect all the tags of calibration database [release/SBN2023A]

### PR DESCRIPTION
**Pull request by jedori0228**
Calibration database through sqlite requires tags. These are specified in multiple fhicls, and must be updated when we have new measurements. I defined a single fhicl file (`fcl/configurations/calibration_database_GlobalTags_icarus.fcl`) which will be included downstream. Tags are groups for each sub-system (TPC/PMT/CRT) in separate "TagSets" fhicls:
- `fcl/configurations/calibration_database_TPC_TagSets_icarus.fcl`
- `fcl/configurations/calibration_database_PMT_TagSets_icarus.fcl`
- `fcl/configurations/calibration_database_CRT_TagSets_icarus.fcl`

In each "TagSets" fhicls, I grouped different combination of tags for different purpose; e.g., in the PMT side, we have "PMT_CalibrationTags_Run1", "PMT_CalibrationTags_Run2_August2023", or "PMT_CalibrationTags_Run2_Sept2023" for different time periouds (done by @mvicenzi in [#629 ](https://github.com/SBNSoftware/icaruscode/pull/629/files))
Then in `fcl/configurations/calibration_database_GlobalTags_icarus.fcl`, we select which "sets" to use and construct "ICARUS_Calibration_GlobalTags" table.

I have tested running "fhicl-dump" on stage0/1/CAFmaker fhicls and saw we get correct tags.

* I updated the tag for `tpc_channelstatus_data` from `v1r1` to `v3r1`. This requires an update in `icarus_data`.

### CODE REVIEW

File: fcl/configurations/calibration_database_PMT_TagSets_icarus.fcl [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# File:    timing_corrections_tags.fcl
+# Author:  M. Vicenzi (mvicenzi@bnl.gov)
```
Comment (mvicenzi) [inline fcl/configurations/calibration_database_PMT_TagSets_icarus.fcl:2]: To be pedantic:
```suggestion
# File:    calibration_database_PMT_TagSets_icarus.fcl
# Author:  M. Vicenzi (mvicenzi@bnl.gov), J. Kim (jaesung.kim.3426@gmail.com)
```

**Review by mvicenzi — APPROVED** [review]
From the PMT side, I have no objections in moving these tag definitions in a central location. I have a pedantic comment below, but I'm pre-approving.

Are you planning to also port this change in `develop`?

**Review by cfarnese — APPROVED** [review]
I take a look in particular to the part related to the TPC! Thanks for the changes! I hope they can be propagated to the present production release and also to the most recent icaruscode release

File: fcl/configurations/calibration_database_PMT_TagSets_icarus.fcl (hunk 1 above)
Comment (jedori0228) [reply fcl/configurations/calibration_database_PMT_TagSets_icarus.fcl]: I updated this, thanks!

**Comment by jedori0228** [comment]
> From the PMT side, I have no objections in moving these tag definitions in a central location. I have a pedantic comment below, but I'm pre-approving.
> 
> Are you planning to also port this change in `develop`?

Yep! I am preparing a copy PR against `develop`.

**Review by varanini — APPROVED** [review]

**Comment by mmrosenberg** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_72_00 SBNSoftware/sbncode@v09_72_00_05 SBNSoftware/sbnanaobj@v09_20_06_02 SBNSoftware/sbndaq-artdaq-core@v1_06_00of0 SBNSoftware/sbnobj@v09_16_00_01 SBNSoftware/icarus_signal_processing@v09_63_00_02 SBNSoftware/icarusalg@v09_72_00_04 SBNSoftware/icarusutil@v09_66_02

**Review by PetrilloAtWork — APPROVED** [review]
Looks good to me.

**Comment by mmrosenberg** [comment]
manually triggered CI tests with e20, c7 qualifiers passed

**Pull request merged**